test/parallel/test-fs-cp-sync-dereference.js

```diff
@@ -15,25 +15,38 @@ const {
   mkdirSync,
   symlinkSync,
   writeFileSync,
+  readFileSync,
+  statSync
 } = require('fs');
 
 const {
   join,
 } = require('path');
 
+const assert = require('assert/strict');
```

Comment:
```suggestion
const assert = require('assert');
```

Using `assert/strict` explicitly is unnecessary here since you're not using any of the non-strict methods.